modules/va_facilities/README.v1.yml

```diff
@@ -37,6 +37,10 @@ info:
     - The list of available services in the `services` element is only applicable to VA health and
       benefits facilities.
 
+    ### Mobile Facilities
+
+    VA has a set of mobile facilities that move regularly within a region. If a facility comes back from this API with `"mobile": "true"`, the latitude and longitude should be considered inaccurate.
```

Comment:
Maybe we should also say that the appropriate way to get the current address is the contact the mobile facility directly.